src/main/java/Duke.java

```diff
@@ -3,6 +3,9 @@
 import java.util.regex.Pattern;
 import java.util.ArrayList;
 import java.io.*;
```

Comment:
Maybe you should consider listing out all the imported features from the java.io library instead of importing the whole package as a whole.